New tabPanel for learner selection and construction

florianfendt opened the pull request:
Quite a huge change.
When this is merged we have a new tabPanel for the learner selection.
Here one can select learners and change their param settings.
Also probability estimation is possible for classif tasks.
In the benchmark panel one can then choose between the preselected learners.
This also applies for the train & predict panel.

Coorsaa commented:
@florianfendt 
looks already pretty good! However, when adding or removing a learner, all hyperpar settings are reset to empty lists.

florianfendt commented:
@Coorsaa 
I fixed the issue by adding couple of actionButtons.
I also implemented some informative error messages when the task is not created yet (which we need to do now with an actionButton, as well)
I'll merge this in now, if you encounter any errors let me know and/or open an issue we can then still fix it later.
